application/src/test/java/org/thingsboard/server/edge/AbstractEdgeTest.java

```diff
@@ -136,11 +138,12 @@ public void setupEdgeTest() throws Exception {
         doPost("/api/admin/jwtSettings", settings).andExpect(status().isOk());
 
         loginTenantAdmin();
-
+        //7 installation messages
```

Comment:
I like this approach with additional comments, but please extend it and add details on what type of messages are expected.
This will simplify development and testing. 
This commend applicable for all cases, where you have:
```
//3 messages
//1 message
//1 message 

```etc.